src/parcels/_core/field.py

```diff
@@ -483,3 +487,34 @@ def _get_positions(field: Field, time, z, y, x, particles, _ei) -> tuple[dict, d
     _update_particles_ei(particles, grid_positions, field)
     _update_particle_states_position(particles, grid_positions)
     return particle_positions, grid_positions
+
+
+def _select_uxinterpolator(da: ux.UxDataArray):
```

Comment:
I can also imagine this function to live in FieldSet, where we parse all the metadata and find out whether the dataset is CF-compliant

Reply:
This is now set in `fieldset.py` and the default interpolator selection in `field.py` has been removed. This also required updating all tests to explicitly pass the `interp_method`.